src/App.tsx

```diff
@@ -0,0 +1,43 @@
+import { Container, Typography, Box, ThemeProvider, Link } from '@mui/material';
+import ProjectList from './components/ProjectList';
+import backgroundImg from './assets/PlaygroundBg.webp'
+import { playgroundTheme } from './styles/playgroundTheme';
+import { useTheme } from '@mui/material/styles';
+import { useMediaQuery } from '@mui/material';
+
+
+function App() {
+const theme = useTheme();
+const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+
+const paddingValue = isMobile ? '42px' : '120px';
+  return (
+    <ThemeProvider theme={playgroundTheme}>
+    <Box
+    component="div"
+    sx={{
+      width: '100vw',
+      height: '100%',
+      minHeight: '100vh',
+      backgroundImage: `url(${backgroundImg})`,
```

Comment:
Text is unreadable until the image loads - especially noticable on slower connection. Add `backgroundColor: 'blue',`